### PR TITLE
Controller

### PR DESCRIPTION
**Pull request by cicadapurp**
add basic routes for quiz and  question

### CODE REVIEW

File: app/controllers/api/choices_controller.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    render json: @choices
+  end
+  
+  def new 
+  end
+
+  def create
+    @choice = Choice.new(choice_params)
+    if @choice.save
+      render json: @article, status: :created, location: //something
```
Comment (spenhappenin) [inline app/controllers/api/choices_controller.rb]: @article?

File: app/controllers/api/choices_controller.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    else
+      render json: @choices.errors, status: :unprocessable_entity
+    end 
+  end
+
+  def update
+    if @choice.update(choice_params)
+
+      render json: @article
```
Comment (spenhappenin) [inline app/controllers/api/choices_controller.rb]: A few more articles here

Comment (pompucha) [reply app/controllers/api/choices_controller.rb]: also fixed this one.

File: app/controllers/api/choices_controller.rb (hunk 1 above)
Comment (pompucha) [reply app/controllers/api/choices_controller.rb]: i fix this one

**Pull request merged**